sdk/resourcemanager/azure-resourcemanager-eventhubs/pom.xml

```diff
@@ -37,13 +37,14 @@
 
   <properties>
     <!-- RMJacoco -->
-    <jacoco.min.linecoverage>0.10</jacoco.min.linecoverage>
-    <jacoco.min.branchcoverage>0.10</jacoco.min.branchcoverage>
+    <jacoco.min.linecoverage>0</jacoco.min.linecoverage>
+    <jacoco.min.branchcoverage>0</jacoco.min.branchcoverage>
     <!-- Configures the Java 9+ run to perform the required module exports, opens, and reads that are necessary for testing but shouldn't be part of the module-info. -->
     <javaModulesSurefireArgLine>
       --add-opens com.azure.resourcemanager.eventhubs/com.azure.resourcemanager.eventhubs=ALL-UNNAMED
       --add-opens com.azure.resourcemanager.storage/com.azure.resourcemanager.storage=ALL-UNNAMED
     </javaModulesSurefireArgLine>
+    <revapi.skip>true</revapi.skip>
```

Comment:
Wonder what breaks here...

Reply:
OK, I saw the breaks are about `Indentity` classes. They are due to our recent codegen fix. We can skip the revapi.
https://dev.azure.com/azure-sdk/public/_build/results?buildId=2725264&view=logs&jobId=b70e5e73-bbb6-5567-0939-8415943fadb9&j=b70e5e73-bbb6-5567-0939-8415943fadb9&t=68a849d6-f425-5f38-d450-7e2c3d1df885